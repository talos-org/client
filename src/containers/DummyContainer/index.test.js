import React from 'react';
import { shallow } from 'enzyme';

import { DummyContainer as Component } from '.';

describe('Component', () => {
  it('should render correctly in "debug" mode', () => {
    const component = shallow(<Component debug />);

    expect(component).toMatchSnapshot();
  });
});
