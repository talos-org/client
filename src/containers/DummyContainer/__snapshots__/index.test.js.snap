// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component should render correctly in "debug" mode 1`] = `
<div>
  Dummy
</div>
`;
